cmd/coder/login.go

```diff
@@ -22,13 +23,13 @@ type loginCmd struct {
 func (cmd loginCmd) Spec() cli.CommandSpec {
 	return cli.CommandSpec{
 		Name:  "login",
-		Usage: "[Coder Enterprise URL]",
+		Usage: "[Coder Enterprise URL eg. http://my.coder.domain/ ]",
 		Desc:  "authenticate this client for future operations",
 	}
 }
 func (cmd loginCmd) Run(fl *pflag.FlagSet) {
 	rawURL := fl.Arg(0)
-	if rawURL == "" {
+	if rawURL == "" || !strings.HasPrefix(rawURL, "http") {
 		exitUsage(fl)
```

Comment:
I tested against http://localhost:8080, http://master.cdr.dev/ and https://master.cdr.dev/ and all seem to work (ie., master.cdr.dev auto-downgrades the login URL to http, which I assume is desired).